packages/paypal-commerce-integration/src/paypal-commerce/paypal-commerce-payment-initialize-options.ts

```diff
@@ -51,7 +51,7 @@ export default interface PayPalCommercePaymentInitializeOptions {
     /**
      * A callback for displaying error popup. This callback requires error object as parameter.
      */
-    onError?(error: Error): void;
+    onError?(error: Error | unknown): void;
```

Comment:
Looks like we can leave only the `unknown` type in this case and remove the `Error`